<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addPolyline</class>
 <widget class="QDialog" name="addPolyline">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>370</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>0</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Polyline Creator&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="shapeDimensions">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>371</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Shape Dimensions</string>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>328</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="point">
       <property name="text">
        <string>Point 1 (x1,y1)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="x_1"/>
     </item>
     <item>
      <widget class="QLineEdit" name="y_1"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>328</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="point_2">
       <property name="text">
        <string>Point 2 (x2,y2)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="x_2"/>
     </item>
     <item>
      <widget class="QLineEdit" name="y_2"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>328</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="point_3">
       <property name="text">
        <string>Point 3 (x3,y3)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="x_3"/>
     </item>
     <item>
      <widget class="QLineEdit" name="y_3"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>328</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="point_4">
       <property name="text">
        <string>Point 4 (x4,y4)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="x_4"/>
     </item>
     <item>
      <widget class="QLineEdit" name="y_4"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>328</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="point_5">
       <property name="text">
        <string>Point 5 (x5,y5)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="x_5"/>
     </item>
     <item>
      <widget class="QLineEdit" name="y_5"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="penProperties">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>301</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Pen Properties</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>136</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="color">
     <item>
      <widget class="QLabel" name="penColor">
       <property name="text">
        <string>Pen Color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="penColorBox">
       <item>
        <property name="text">
         <string>---</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>White</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Black</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Red</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Green</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Blue</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cyan</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Magenta</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Yellow</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gray</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>171</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="style">
     <item>
      <widget class="QLabel" name="penStyle">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pen Style&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="penStyleBox">
       <item>
        <property name="text">
         <string>---</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NoPen</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SolidLine</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DashLine</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DotLine</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DashDotLine</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DashDotDotLine</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>166</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="capstyle">
     <item>
      <widget class="QLabel" name="penCapStyle">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pen Cap Style&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="penCapStyleBox">
       <item>
        <property name="text">
         <string>---</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>FlatCap</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SquareCap</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RoundCap</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>165</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="penJoinStyle">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pen Join Style&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>---</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MiterJoin</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BevelJoin</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RoundJoin</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>105</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="penWidth">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pen Width&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="penWidthBox">
       <property name="minimumSize">
        <size>
         <width>42</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
